kubernetes/operationcode_backend/deployment.yml

```diff
@@ -27,37 +27,52 @@ spec:
             secretKeyRef:
               name: backend-secrets
               key: airtable_api_key
+        - name: SLACK_LEGACY_ADMIN_TOKEN 
```

Comment:
PR says adding `POSTGRES_USER`, where'd this come from?

Reply:
Initially it was just going to be this. But these variables ended up being required to get the slack invites working. These environment variables were initially stored in a way that made it not obvious they were used. When we deactivated them we forgot they were still in use.